app/assets/stylesheets/application.scss

```diff
@@ -13,4 +13,20 @@
  *= require_tree .
  *= require_self
  */
- @import "bootstrap";
+@import "bootstrap";
+/* Base */
```

Comment:
The 'Base' section should be placed at the top of the file, before any other imports or styles.